src/nvim/api/vim.c

```diff
@@ -2142,6 +2157,17 @@ Dictionary nvim_eval_statusline(String str, Dict(eval_statusline) *opts, Error *
         fillchar = fillchar_status(&attr, wp);
       }
     }
+    if (use_statuscol) {
+      HlPriAttr num, line, cul;
```

Comment:
These variables being uninitialized may cause a static analyzer warning.

Reply:
Right, the question is if we want to make some `drawline.c` functions non-`static` in order to be able to initialize these properly, or just silence the potential warnings.

Reply:
The refactoring required for the relevant functions to be used in `nvim_eval_status()` would defeat the purpose of the recently added `winlinevars_T`. Better inline whats needed if we want `opts->highlights` to return the proper attrs I guess.